File: fizzbuzz.go
package main

import "fmt"

func main() {
	for i := 1; i <= 15; i++ {
		var output = ""
		if i % 3 == 0 {
			output += "fizz"
		}
		if i % 5 == 0 {
			output += "buzz"
		}
		if output != "" {
			fmt.Println(output);
		} else {
			fmt.Println(i);
		}
	}
}

File: fizzbuzz.vb
module fizzbuzz
    sub main()
        dim i as integer
        for i = 1 to 15
	    dim output as string = ""
            if i mod 3 = 0 then
                output += "fizz"
            end if
            if i mod 5 = 0 then
		output += "buzz"
            end if
	    if output = ""
                output = trim(str(i))
            end if
            console.writeline(output)
        next
    end sub
end module
